src/containers/Navbar/Navbar.container.tsx

```
@@ -0,0 +1,89 @@
+
+export const Navbar = () => {
+    const navigate = useNavigate();
+    const isMobile = useMediaQuery((theme) => theme.breakpoints.down('md'));
+    /**
+     * Handle search bar submission.
+     *
+     * Navigates to the product page if the input value is not null.
+     */
+    function handleSubmit(value: string | null) {
+        if (value) {
+            void navigate('/product/' + value);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)